mesa/cookiecutter-mesa/CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# Change Log
+All notable changes to this project will be documented in this file.
```

Comment:
@drewrey this changelog file is for the cookiecutter template itself. It is not necessary, though, since the template is encapsulated in the main mesa repo for now.

Note: We could make a standalone repo for the cookiecutter template, but then the mesa-init command would have this external repo as a dependency.

@jackiekazil we could also add a per-model history file to the cookiecutter template. I will defer that addition for now.

In summary there are at least 3 places to track history:
1. **mesa repo itself**, which is already covered by the [HISTORY.rst](https://github.com/projectmesa/mesa/blob/master/HISTORY.rst)
2. **cookiecutter template**, which is what this changelog at `mesa/cookiecutter-mesa/CHANGELOG.md` or `mesa/cookiecutter-mesa/HISTORY.rst` would track
3. **the new mesa project** created by the mesa-init command, i.e. the user's own model could have a history file. this would be inside of the `{{cookiecutter.title}}` directory